job.go

```diff
@@ -315,3 +317,17 @@ func (j *AuroraJob) Container(container Container) Job {
 
 	return j
 }
+
+// Set the Tier for the Job.
+func (j *AuroraJob) Tier(tier *string) Job {
```

Comment:
Let's make this function take a concrete type as an argument and then assign the address of it to the `.Tier` field. That way we don't have to create a pointer to pass to this function.